cmp/grid/Grid.scss

```diff
@@ -192,3 +192,22 @@
 .xh-grid-tooltip {
   position: absolute;
 }
+
+.xh-grid-autosize-canvas {
+  position: absolute;
+  display: none;
+}
+
+.xh-grid-autosize-cell {
+  top: var(--xh-grid-tree-indent-px);
```

Comment:
Why `top` here for the indent?  I am sure this is as intended, but seems worth a comment as it threw me for a loop (I would have assumed `left` since we're all about measuring width / indenting)

Reply:
We're not *actually* using it directly - this is a dummy style rule, used only as a way to trampoline the `--xh-grid-tree-indent-px` value in a way that it can be brought into JS via `window.getComputedStyle()` (it uses a calc internally). Will comment as such to explain.